cloudflare-rails.gemspec

```diff
@@ -32,7 +32,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "appraisal"
 
   spec.add_dependency "httparty"
-  spec.add_dependency "rails", ">= 5.0", "< 6.2.0"
+  spec.add_dependency "railties", ">= 5.0", "< 6.2.0"
```

Comment:
Wouldn't this disallow minor Rails updates?

Maybe something like this could work instead, so that it's possible to upgrade to 6.2.0 in the future:

`spec.add_dependency "rails", ">= 5.0", "< 7"`

Reply:
in general i've been locking at minor releases because i wanted to guarantee compatibility (iirc there were changes between 4.1 and 4.2 that broke this gem). i've been pretty quick to release new versions when rails does so hopefully this doesn't block anyone for too long.

Reply:
also in this specific case there's no 6.2 so things should be fine:

https://github.com/rails/rails/commit/1b455e2e9d6937d4107e19cb32e2f98aa08886b9

i've also got a branch working on 7.0.0.alpha so i should be ready to go when that drops. 

https://github.com/modosc/cloudflare-rails/pull/36